src/App.css

```diff
@@ -155,4 +155,39 @@ input[type='text'] {
     height: 300px;
     object-fit: contain;
   }
-}
\ No newline at end of file
+}
+/* for movie and year */
+.container {
```

Comment:
This container class styling overrides the previous container styling. It is not needed, please remove it